docs/conf.py

```diff
@@ -42,7 +42,11 @@
 
 autoapi_type = "python"
 autoapi_dirs = ["../src"]
-autoapi_ignore = ["*/__main__.py", "*/_version.py"]
+autoapi_ignore = [
+    "*/__main__.py",
+    "*/_version.py",
+    "*/datatypes.py",
```

Comment:
This line is a temporary fix like we discussed. I have created a separate issue for this - #31.